src/neo/SmartContract/ApplicationEngine.cs

```diff
@@ -20,11 +20,17 @@ public partial class ApplicationEngine : ExecutionEngine
         private readonly bool testMode;
         private readonly List<NotifyEventArgs> notifications = new List<NotifyEventArgs>();
         private readonly List<IDisposable> disposables = new List<IDisposable>();
+        private readonly List<StorageKey> updatedKeys = new List<StorageKey>();
 
         public TriggerType Trigger { get; }
         public IVerifiable ScriptContainer { get; }
         public StoreView Snapshot { get; }
         public long GasConsumed { get; private set; } = 0;
+
+        /// <summary>
+        /// RecyclingRewardGas shows how many gas should be recycled as a reward in systemfee
```

Comment:
Perhaps it is better to change to:

`RecyclingRewardGas shows how many gas should be recycled as a reward`,

Remove the sys_fee part because perhaps sys_fee are going to be burned.